src/status_im/chat/models/input.cljs

```diff
@@ -235,7 +235,8 @@
                       :params     [{:id message-id :text text}]
                       :js-response true
                       :on-error #(log/error "failed to edit message " %)
-                      :on-success #(re-frame/dispatch [:sanitize-messages-and-process-response %])}]}
+                      :on-success #(re-frame/dispatch [:sanitize-messages-and-process-response %])}]
+    :set-input-text [(:current-chat-id db) ""]}
```

Comment:
why not `cancel-message-edit` ? it seems we need to clear more ?

Reply:
if we can re-use cancel-message-edit and it works, I would probably use that, as it's very fiddly to clean the input text

Reply:
thanks, will try

Reply:
seems to work fine